integration-tests/src/tests/client/features/simple_test_loop_example.rs

```diff
@@ -173,36 +155,28 @@ fn test_client_with_simple_test_loop() {
         None,
         Default::default(),
         None,
-        builder.sender().into_wrapped_multi_sender::<SyncJobsSenderForClientMessage, _>(),
+        sync_jobs_adapter.as_multi_sender(),
         Box::new(builder.sender().into_future_spawner()),
     )
     .unwrap();
 
-    let data =
-        TestData { dummy: (), client: client_actor, sync_jobs: sync_jobs_actor, shards_manager };
+    let sync_jobs_sender = builder.register_actor(sync_jobs_actor);
+    let shards_manager_sender = builder.register_actor(shards_manager);
+    let client_sender = builder.register_actor(client_actor);
+
+    shards_manager_adapter.bind(shards_manager_sender);
+    sync_jobs_adapter.bind(sync_jobs_sender);
+    client_adapter1.bind(client_sender.clone());
```

Comment:
so even if there is a single client we need N adapters for N references of that adapter? is this to respect the borrowing rules or something like that?

Reply:
Surprisingly it works with just one copy of LateBoundSender! Will change.